From 303b2ef181a63734fb6b8042d7550492cb9efc5d Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Wed, 8 Oct 2014 02:11:50 +0900
Subject: [PATCH] Add install rule for jenkins.ros.org

---
 CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 182f7b0..4cccb4c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -556,10 +556,10 @@ endif(ENABLE_CPACK)
 
 #if catkin environment
 string(REGEX MATCH "catkin" need_catkin $ENV{_})
-if(need_catkin)
+#if(need_catkin)
   install(FILES package.xml DESTINATION share/openhrp3/)
   install(DIRECTORY test DESTINATION share/openhrp3 USE_SOURCE_PERMISSIONS)
   install(CODE "execute_process(COMMAND cmake -E create_symlink ../../bin/openhrp-model-loader openhrp-model-loader WORKING_DIRECTORY \${CMAKE_INSTALL_PREFIX}/share/openhrp3)")
   install(CODE "execute_process(COMMAND cmake -E create_symlink ../../bin/export-collada       export-collada       WORKING_DIRECTORY \${CMAKE_INSTALL_PREFIX}/share/openhrp3)")
-endif()
+#endif()
 
-- 
1.7.9.5

