From 96284405df7989dfe84908767fd96736b0bf6688 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Mon, 7 Aug 2017 13:32:32 +0900
Subject: [PATCH 3/3] isNoUnknownAccelMode: c1/M11 is inistialized as 0 size
 #126

---
 hrplib/hrpModel/ForwardDynamicsCBM.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hrplib/hrpModel/ForwardDynamicsCBM.cpp b/hrplib/hrpModel/ForwardDynamicsCBM.cpp
index 80b3428..d4639da 100644
--- a/hrplib/hrpModel/ForwardDynamicsCBM.cpp
+++ b/hrplib/hrpModel/ForwardDynamicsCBM.cpp
@@ -695,6 +695,8 @@ void ForwardDynamicsMM::solveUnknownAccels(Link* link, const Vector3& fext, cons
 
 void ForwardDynamicsMM::solveUnknownAccels(const Vector3& fext, const Vector3& tauext)
 {
+    if(isNoUnknownAccelMode){ return ; }
+
     if(unknown_rootDof){
         c1.head(3)      = fext;
         c1.segment(3,3) = tauext;
-- 
2.17.1

