From c4fb556bfaa0b578f0422149444168cef5a51c5a Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Fri, 13 Feb 2015 19:52:02 +0900
Subject: [PATCH 3/4] add .catkin to CMakeLists.txt

---
 CMakeLists.txt | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2571b6c..e8cf7b3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -572,4 +572,30 @@ execute_process(COMMAND cmake -E make_directory share WORKING_DIRECTORY \$ENV{DE
 execute_process(COMMAND cmake -E create_symlink ../../OpenHRP-3.1 share/OpenHRP-3.1 WORKING_DIRECTORY \$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/share/openhrp3)
 ")
 endif()
+string(REGEX MATCH "catkin" need_catkin "$ENV{_}")
+if(need_catkin)
+  install(CODE "
+## this is tricky force write catkin marker file
+set(_catkin_marker_file \"\${CMAKE_INSTALL_PREFIX}/.catkin\")
+# check if the develspace marker file exists yet
+if(EXISTS \${_catkin_marker_file})
+  file(READ \${_catkin_marker_file} _existing_sourcespaces)
+  if(_existing_sourcespaces STREQUAL \"\")
+    # write this sourcespace to the marker file
+    file(WRITE \${_catkin_marker_file} \"${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}\")
+  else()
+    # append to existing list of sourcespaces if it's not in the list
+    list(FIND _existing_sourcespaces \"${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}\" _existing_sourcespace_index)
+    if(_existing_sourcespace_index EQUAL -1)
+      file(APPEND \${_catkin_marker_file} \";${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}\")
+    endif()
+  endif()
+else()
+  # create a new develspace marker file
+  # NOTE: extra care must be taken when running multiple catkin jobs in parallel
+  #       so that this does not overwrite the result of a similar call in another package
+  file(WRITE \${_catkin_marker_file} \"${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}\")
+endif()
+")
+endif()
 
-- 
1.9.1

