From bd10a111533956feb07a0d5d9c0876b965fa2fbe Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Fri, 13 Feb 2015 20:13:36 +0900
Subject: [PATCH 4/5] fix for cmake error on new builddfirm

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e8cf7b3..89e9f5a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -555,7 +555,7 @@ if(ENABLE_CPACK)
 endif(ENABLE_CPACK)
 
 #if catkin environment
-string(REGEX MATCH "catkin" need_catkin $ENV{_})
+string(REGEX MATCH "catkin" need_catkin "$ENV{_}")
 if(need_catkin OR "${CATKIN_BUILD_BINARY_PACKAGE}")
   install(FILES package.xml DESTINATION share/openhrp3/)
   install(DIRECTORY test DESTINATION share/openhrp3 USE_SOURCE_PERMISSIONS)
-- 
1.9.1

